### ISSUE TITLE
[A11y] Work Show Page-CPD Child - Tables

### ISSUE THREAD

**Issue by jechols** [OregonDigital/OD2#2027]
### Operating System

Windows

### Browser

Edge

### Affected page

Work Show

### Admin user?

Yes

### Issue Description

https://staging.oregondigital.org/concern/documents/df727057n

Very similar to #2018, but the child view has its own problems, one of which I believe is a definite WCAG failure.

Like #2018, the "WORKS AND FILES" title cells could / should be row headers for the table. Not a failure on its own, but combined with the next issue it is.

The thumbnail's accessible name is "file details". This would be fine if there were a row header because the row header would provide context. It's not ideal as an accessible name, but it's actually describing exactly what it links you to **if** it's given context. One could argue to just add invisible text to it, but then you lose out on the value of having row-level context (or else you duplicate it), and you have magic hidden text, which is always best to avoid when there's a way to associate visible text instead.

### Suggested Fix

Ideally, just make the title cells a row header.

### Can anybody QA or does this need an expert?

Needs expert

### QA Steps

This is easiest to test with a screen reader, but somebody very detail-oriented can verify the table with HTML and tools like the Firefox accessibility inspectors.

**Comment by jechols** [comment]
Merged this into #2018
